<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ESP_Wifi_Config_Window</class>
 <widget class="QMainWindow" name="ESP_Wifi_Config_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="wifi_station_setting_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>WiFi Station Setting</string>
    </property>
    <widget class="QLabel" name="file_path_lable">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>File Path</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="basic_setting_groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>251</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Basic Setting</string>
     </property>
     <widget class="QLineEdit" name="gateway_lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="subnet_lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="static_ip_lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="static_ip_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Static IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="gateway_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Gateway</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gateway_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subnet_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="static_ip_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="subnet_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Subnet</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="dhcp_enable_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DHCP Enable</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ssid_lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="password_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ssid_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="ssid_lable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>SSID</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dhcp_enable_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="connection_timeout_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Timeout</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="connection_timeout_lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="connection_timeout_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="submit_all_pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit All</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file_path_lineEdit">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="file_load_pushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="file_save_pushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="configuration_data_plainTextEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>251</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="basic_setting_to_text">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="text_to_basic_setting">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>591</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Quick Command</string>
     </property>
     <widget class="QPushButton" name="get_properties_pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Properties</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wifi_connect_pushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>WiFi Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wifi_disconnect_pushButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>WiFi Disconnect</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="device_reboot_pushButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Device Reboot</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="com_port_groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>401</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>COM Port</string>
    </property>
    <widget class="QComboBox" name="port_list_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="baudrate_comboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="port_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLabel" name="baudrate_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BaudRate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect_pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="connect_state_label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnected</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="data_from_com_port_plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>381</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
